src/dbnode/storage/shard.go

```diff
@@ -1451,18 +1445,16 @@ func (s *dbShard) FetchBlocksMetadata(
 ) (block.FetchBlocksMetadataResults, *int64, error) {
 	switch s.opts.SeriesCachePolicy() {
 	case series.CacheAll:
-	case series.CacheAllMetadata:
 	default:
-		// If not using CacheAll or CacheAllMetadata then calling the v1
+		// If not using CacheAll then calling the v1
 		// API will only return active block metadata (mutable and cached)
 		// hence this call is invalid
 		return nil, nil, fmt.Errorf(
 			"fetch blocks metadata v1 endpoint invalid with cache policy: %s",
 			s.opts.SeriesCachePolicy().String())
 	}
 
-	// For v1 endpoint we always include cached blocks because when using
-	// CacheAllMetadata the blocks will appear cached
+	// For v1 endpoint we always include cached blocks
```

Comment:
Should be able to remove the V1 endpoint code at this point too /cc @robskillington

Reply:
delete all the things

Reply:
lets definitely def delete all V1 code/tests. Don't do the rename tho - the 1-1 parity between go methods and the thrift code is nice. Plus it means if we ever add v3 or w/e it'd be a lot more explicit